packages/repository-json-schema/src/build-schema.ts

```diff
@@ -10,9 +10,11 @@ import {
 } from '@loopback/repository';
 import {includes} from 'lodash';
 import {Definition, PrimitiveType} from 'typescript-json-schema';
-import {MetadataInspector} from '@loopback/context';
+import {MetadataInspector, MetadataAccessor} from '@loopback/context';
 
-export const JSON_SCHEMA_KEY = 'loopback:json-schema';
+export const JSON_SCHEMA_KEY = MetadataAccessor.create<JsonDefinition>(
```

Comment:
Why is there no decorator type passed here in the `<>`?

Reply:
If missing, any decorator type is fine. There are valid cases that the same string is used for both class and member decorators.